main.cpp

```diff
@@ -80,5 +7,7 @@ int main() {
     testPairToNumber(TelCoColorCoder::BLACK, TelCoColorCoder::ORANGE, 12);
     testPairToNumber(TelCoColorCoder::VIOLET, TelCoColorCoder::SLATE, 25);
 
+    //print reference manual for wiring personnel
+    std::cout<<printRefManualData();
```

Comment:
Good decision to split the formation and printing of the reference manual. Can you think of any asserts on the return value, so that the burden of manual testing is reduced?